android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -363,7 +363,7 @@ private void sendEvent(ReactContext reactContext,
   }
 
   @ReactMethod
-  public void isHeadsetPluggedIn(Promise promise) {
+  public void isHeadsetPlugged(Promise promise) {
     boolean headphonesLocated = isHeadsetPlugged();
```

Comment:
Which function `isHeadsetPlugged` calls? `public void isHeadsetPlugged` or 'priavate Bool isHeadsetPlugged`?

Reply:
Oh! These two methods have different arguments. It can be distinguishable.